src/App.vue

```diff
@@ -1,28 +1,38 @@
 <template>
   <v-app>
     <v-main>
-      <div v-if="!widgetStore.editingMode && !showMainMenu && interfaceStore.mainMenuStyleTrigger === 'center-left'">
+      <div
+        v-if="
+          !widgetStore.editingMode &&
+          !interfaceStore.showMainMenu &&
+          interfaceStore.mainMenuStyleTrigger === 'center-left'
+        "
+      >
```

Comment:
We did raise the `eslint` rule for line chars right? Is it still giving us trouble?

Reply:
Prettier config 'printWidth' was back to 120. Fixed